### PR TITLE
Update additional_software.md (punctuation)

### PR DESCRIPTION
**Pull request by SergeCroise**
- punctuation
- a ncurses-style

#### Author checklist (Completed by original Author)
- [x] Good fit for the Rocky Linux project? Title and Author Metatags inserted ?
- [x] If applicable, steps and instructions have been tested to work
- [x] Initial self-review to fix basic typos and grammar completed

#### Rocky Documentation checklist (Completed by Rocky team) 
- [ ] 1st Pass (Document is good fit for project and author checklist completed)
- [ ] 2nd Pass (Technical Review - check for technical correctness) 
- [ ] 3rd Pass (Detailed Editorial Review and Peer Review)
- [ ] Final approval (Final Review)

### CODE REVIEW

File: docs/books/nvchad/additional_software.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ RipGrep is needed for recursive searches with `:Telescope`.
 
 ## Lazygit
 
-[LazyGit](https://github.com/jesseduffield/lazygit) is an ncurses-style interface that allows you to perform all `git` operations in a more user-friendly way. It is required by the _lazygit.nvim_ plugin. This plugin makes it possible to use LazyGit directly from NvChad, it opens a floating window from where you can perform all operations on your repositories, thus allowing you to make all changes to the _git repository_ without leaving the editor.
+[LazyGit](https://github.com/jesseduffield/lazygit) is a ncurses-style interface that allows you to perform all `git` operations in a more user-friendly way. It is required by the _lazygit.nvim_ plugin. This plugin makes it possible to use LazyGit directly from NvChad, it opens a floating window from where you can perform all operations on your repositories, thus allowing you to make all changes to the _git repository_ without leaving the editor.
```
Comment (SergeCroise) [inline docs/books/nvchad/additional_software.md:64]: not sure what is the right article
a ncurses-style or
an ncurses-style

**Comment by SergeCroise** [comment]
Hello @ambaradan , @sspencerwire , @wsoyinka
please review!

File: docs/books/nvchad/additional_software.md (hunk 1 above)
Comment (wsoyinka) [reply docs/books/nvchad/additional_software.md:64]: an is correct here. thanks

**Review by wsoyinka — APPROVED** [review]

**Comment by wsoyinka** [comment]
Thanks @SergeCroise :-) !

**Pull request merged**